### DISCUSSION TITLE
scrolling

### DISCUSSION THREAD

**Discussion by ranran2121** [prisma/prisma#17368] [Q&A]
Hi,

is there any way to have control on the scrolling?

**Comment by nurul3101** [comment]
Hi @ranran2121 👋 
Can you please elaborate on your use case?
Maybe share the schema file and corresponding query?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
